reana_server/config.py

```diff
@@ -106,12 +112,19 @@ def _(x):
 )
 
 CERN_APP_CREDENTIALS = dict(
-    consumer_key='CHANGE_ME',
-    consumer_secret='CHANGE_ME',
+    consumer_key=CERN_CONSUMER_KEY,
+    consumer_secret=CERN_CONSUMER_SECRET,
 )
 
 DEBUG = True
 
 SECURITY_PASSWORD_SALT = 'security-password-salt'
 
 SECURITY_SEND_REGISTER_EMAIL = False
+
+# Gitlab Application configuration
+# ================================
+GITLAB_APP_ID = os.getenv('GITLAB_APP_ID', 'CHANGE_ME')
```

Comment:
For all GitLab ones, `REANA_GITLAB_OAUTH` prefix. For example, `REANA_GITLAB_OAUTH_APP_ID`. And you can move them to the upper section since this has nothing to do with Invenio.